Sources/ArcGISToolkit/Components/Popups/PopupView.swift

```diff
@@ -89,11 +88,14 @@ public struct PopupView: View {
                     Button(action: {
                         isPresented?.wrappedValue = false
                     }, label: {
-                        Image(systemName: "xmark.circle")
-                            .foregroundColor(.secondary)
-                            .padding([.top, .bottom, .trailing], 4)
+                        Image(systemName: "xmark")
```

Comment:
I know that this is somewhat beyond the scope of this PR, but not entirely. According to the documentation, we should be using a different initializer:

> ![image](https://github.com/user-attachments/assets/9f478b31-9bf5-439e-95d7-003cc52026c2)

Reply:
Ok, I can make that change here but this seems like something that should be addressed across the toolkit. Can you make an issue for it?

Reply:
Yes, I can make an issue.

Reply:
#942